.github/workflows/ci-disable-gtest.yml

```diff
@@ -73,9 +73,7 @@ jobs:
 
     - name: Prepare libavif (cmake)
       run: >
-        mkdir build && cd build
-
-        cmake .. -G Ninja
+        cmake -G Ninja -S ./ -B build
```

Comment:
The `/` after `.` is not necessary. We don't have `/` after `build`.

Reply:
Fixed in https://github.com/AOMediaCodec/libavif/pull/2176